Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
@raburton, 
copied the Basic_rBoot Makefile and the Makefile-user.mk  setting RBOOT_ENABLED ?= 0

and keep getting : 
c:/tools/Sming/Sming/SmingCore/Network/rBootHttpUpdate.h:14:23: fatal error: rboot-api.h: No such file or directory
 #include <rboot-api.h>
                       ^

Something broke somewhere :D 
Any hint?